src/services/users.service.ts

```diff
@@ -225,10 +225,15 @@ export class UsersService {
   }: {
     ownerId: string;
     tierId: string;
-    maxSpaceBytes: number;
-    seats: number;
+    maxSpaceBytes?: number;
+    seats?: number;
   }): Promise<void> {
     const jwt = signToken('5m', this.config.DRIVE_NEW_GATEWAY_SECRET);
+    let totalMaxSpaceBytes: number | undefined = undefined;
```

Comment:
Why not `let totalMaxSpaceBytes = maxSpaceBytes` here? The param is also optional and if it set, it will go inside the if

Reply:
If it not being used, it should not be passed I assume, but this works also